tests/driver/index.test.ts

```diff
@@ -355,6 +355,17 @@ describe('Driver based tests', async () => {
             );
         });
 
+        it('handles listCollections()', async () => {
+            const personSchema = new mongooseInstance.Schema({
+                name: { type: String, required: true }
+            });
+            const Person = mongooseInstance.model('Person', personSchema);
+            await Person.init();
+            await Person.deleteMany({});
```

Comment:
what does deleteMany do here?

Reply:
Empty the collection, because there's other documents in this collection from other tests